sktime/datatypes/_panel/_check.py

```diff
@@ -190,6 +190,14 @@ def check_pdmultiindex_panel(obj, return_metadata=False, var_name="obj", panel=T
         msg = f"{var_name} must have a MultiIndex, found {type(obj.index)}"
         return _ret(False, msg, None, return_metadata)
 
+    # check to delineate from nested_univ mtype (Hierarchical)
+    # pd.DataFrame mtype allows object dtype,
+    # but if we allow object dtype with Panel entries,
+    # the mtype becomes ambiguous, i.e., non-delineable from nested_univ
```

Comment:
I just copy pasted your comment and tried to edit two terms, but I'm certain this is wrong. So, please review and change.

Reply:
that should work though, let's see what the tests say